src/cmd/manage_nsfs.js

```diff
@@ -206,6 +207,7 @@ async function add_bucket(data) {
     // for validating against the schema we need an object, hence we parse it back to object
     nsfs_schema_utils.validate_bucket_schema(JSON.parse(data_json));
     await native_fs_utils.create_config_file(fs_context, buckets_dir_path, bucket_conf_path, data_json);
+    manage_nsfs_events_utils.create_event('BUCKET', 'BUCKET_CREATED', data.name, {bucket_name: data.name});
     write_stdout_response(ManageCLIResponse.BucketCreated, data_json);
```

Comment:
It looks like we missed some events, mainly error events,
I think that we might want to consider calling the create_event() function from 
write_stdout_response() according to the response code and from throw_cli_error() according to the error we got.
this way we won't miss existing events, I would add a mapping between responses to errors to events
and if the event exists we will call create_event()

Reply:
@romayalon do we need to have an event for all the error scenarios, some CLI errors due to invalid flags and values, And do not point to the state of the system. if we call create_event() from write_stdout_response() it will create an event even for simple status checks and bucket and account listings. I don't see how this will help the monitoring team. WDYT?

Reply:
ofc not, I agree that there are CLI failures that are definetly should not create an event.
I would create an event only to the errors mapped to the events we already have ( for example ManageCLIResponse.BucketCreated is mapped to 'BUCKET_CREATED' event)

Reply:
oh.. ok, Sure will do that